TurkishFilter/sections/specific.txt

```diff
@@ -3424,7 +3424,7 @@ karabuknethaber.com##iframe[width="940"]
 vatandanhaber.com##img[alt$="Magazingo.com"]
 afyonhaber.com##img[alt*="web-banner"]
 ucretsizfilmindir.net##img[alt="Premium Üye OL"]
-akcagazete.com,balikesirhaberajansi.com,kurtulusgazetesi.com.tr##img[alt="Reklam"]
+mavididim.com.tr,akcagazete.com,balikesirhaberajansi.com,kurtulusgazetesi.com.tr##img[alt="Reklam"]
```

Comment:
On my end it is `alt="reklam"`, so I think it should be:
`mavididim.com.tr##img[alt="reklam"]`

<details><summary>Screenshot</summary>

![image](https://user-images.githubusercontent.com/29142494/61790643-136efe00-ae18-11e9-86f0-b28a13293e17.png)

</details>